lib/filesystem.py

```diff
@@ -1294,10 +1284,8 @@ def validate_strict(self):
                      % (msg_prefix, img), ch.BUG_REPORT_PLZ)
 
    def version_read(self):
-      # While support for storage v1 was dropped some time ago, let’s at least
-      # retain the ability to recognize it.
       if (not os.path.isfile(self.version_file)):
-         return 1
+         ch.FATAL('Expected file "%s" not found' % self.version_file)
```

Comment:
What if we instead omit this whole check, and add the version file to `valid_p` per #1147, which you would then add to this PR?